package/network/services/hostapd/Makefile

```diff
@@ -7,13 +7,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=hostapd
-PKG_RELEASE:=32
+PKG_RELEASE:=33
```

Comment:
As `PKG_SOURCE_DATE` is bumped you should reset `PKG_RELEASE` to `1` or (preferably) use `$(AUTORELEASE)`

Reply:
Thank you @dangowrt!
PKG_RELEASE:=$(AUTORELEASE) it is!
This is changed now.
Let me know if you see something else.